### DISCUSSION TITLE
大佬考虑在gitee上建立镜像吗

### DISCUSSION THREAD

**Discussion by swift-lee** [Richasy/Bili.Uwp#71] [想法]
国内GitHub访问速度和稳定性众所周知，虽然gitee最近也开始限速了，但是至少他稳🤣
用action可以很方便的自动push到GitHub：[https://github.com/TobKed/github-forks-sync-action](https://github.com/TobKed/github-forks-sync-action)
用起来也很简单，在gitee里添加一个新的ssh公钥，然后在secret里储存GitHub的access token和gitee的ssh公钥对应的私钥，就ok了
如果没记错的话还可以选择自动push的trigger，可以是在GitHub仓库发生commit时push，也可以是定时push，我自己维护的仓库镜像是2小时一push
（补：release不能自动同步，需要手动发布，这点比较烦人

**Comment by swift-lee** [comment]
（顺便一提，我一直在官方uwp用户群里潜水，至少官方那几个写uwp的人好像完全不知道大佬写过的这两个版本

**Reply by Richasy** [reply]
我也在群里潜水，233😆

**Comment by Richasy** [comment]
我希望问题和讨论可以集中在Github进行统一的处理，发在Gitee我可能没有精力来管

**Reply by swift-lee** [reply]
好吧，理解:-D
